docs/nlp.markdown

```diff
@@ -0,0 +1,43 @@
+# Natural Language Analysis
+
+The QuantGov library includes a number of built-in utilities to perform some
+natural language analysis on a corpus. These can be run using the
+`quantgov nlp` set of commands. All nlp analyses are run at the document level,
+and output a csv. By default, results are printed to standard output, but an
+output file may be specified with the `-o` flag.
+
+For example, the command
+
+``` {.bash}
+quantgov nlp count_words corpora/mycorpus -o wordcount.csv
+```
+
+would produce a word count for the corpus contained in the folder
+`corpora/mycorpus` and save it in the file `wordcount.csv`. The output will
+include one column per index label, with the index label name as a header, and
+then the results of the analyses. The results of `nlp` analyses may therefore
+be merged with any data or statistical analysis software to create a combined
+dataset.
+
+Note that not all analyses may be aggregated above the document level; users
+should familiarize themselves with the metrics produced and use them
+appropriately.
+
+## Available Analyses
+
+-   **count\_words**: Count the number of words in a document. The regular
```

Comment:
I am not sure what you mean by the "regular expression for a word". Maybe clarify what that regular expression looks to match.

Reply:
Clarified.